jax_distribution/wrappers/jumanji.py

```diff
@@ -103,3 +121,65 @@ def observation_spec(self) -> specs.Spec[Observation]:
             "step_count",
         )
         return self._env.observation_spec().replace(step_count=step_count)
+
+
+class RwareMultiAgentWithGlobalStateWrapper(Wrapper):
```

Comment:
Seems like we're going to need to do this for each env, so in future I assume this would go in `wrappers/jumanji/rware.py`?

Reply:
yes agree!